macros/SpecName.ejs

```diff
@@ -92,8 +92,8 @@ var specList = {
         url  : "https://tools.ietf.org/html/draft-west-cookie-prefixes-05"
     },
     'CORS':{
```

Comment:
I've changed spec references on pages from CORS to Fetch. We should remove the CORS entry here.